.travis.yml

```diff
@@ -45,6 +45,7 @@ test_job: &test_job
     - go mod download
   install:
     - go build ./...
+    - RUN apk update && apk add --no-cache openssl && rm -rf "/var/cache/apk/*"
```

Comment:
Our .travis.yml is synced from https://github.com/pion/.goassets/. We'd need a way to inject that snippet somehow.

Reply:
CI runs on Ubuntu, bionic to be precise. So we'd want to see this instead:

```yaml
before_install:
  - sudo apt-get -y install openssl
addons:
  apt:
    update: true
```

https://docs.travis-ci.com/user/installing-dependencies/

Reply:
On pion/webrtc, I have added a repository specific E2E test using GitHub Actions.
https://github.com/pion/webrtc/pull/1007/files

In this way, e2e test code is disabled on Travis,
https://github.com/pion/webrtc/pull/1007/files#diff-783e65d8d8d51bbdfd38558f35cca4caR1
and run e2e test in Docker on GitHub Actions.
https://github.com/pion/webrtc/pull/1007/files#diff-11f78696b5b4b75c7b687a8df25f5951R9-R12

Reply:
I've removed my changes in `.travis.yml` to use the solution proposed by @at-wat